files/copytoclipboard/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Sudheer Kaku"
+github = "https://github.com/sudheerkaku/CopyToClipboard"
+homepage = "http://codebreaker.io/"
```

Comment:
Any reason why its not merged yet? Should I upload the js files or will it be pulled automatically?